deepforest/cascade.py

```diff
@@ -28,6 +28,7 @@ def _get_predictor_kwargs(predictor_kwargs, **kwargs) -> dict:
 
 def _build_classifier_predictor(
     predictor_name,
+    criterion,
```

Comment:
This parameter will not be used when the predictor is GBDTs, right? I think it would be better to give it a default value.

Reply:
if I read it correctly, GBDT doesn't support one of them (mae, mse), maybe we can add a note to the predictor section doc string to docuement the default value for GBDTs, and have let users configure it by predictor_kwarg.

this criterion uses the default value from CascadeForest init, it will only get used by randomforest, maybe we should separate this. e.g. criterion is only for cascade layers, the predictor needs to configure differently

Reply:
My bad, forget that this function is called from `CascadeForest`. I will do some experiments to see if there is a need to set another criterion for the forest predictor ;-)